BepInEx.Templates/templates/BepInEx6.PluginTemplate.Resonite/.template.config/template.json

```diff
@@ -1,37 +1,185 @@
 {
   "$schema": "http://json.schemastore.org/template",
-  "author": "BepInEx",
-  "classifications": [ "BepInEx", "BepInEx 6" , "Plugin" , "CoreCLR", ".NET Core", "Resonite" ],
+  "author": "ResoniteModding",
+  "classifications": [ "BepInEx", "BepInEx 6" , "Plugin" , "CoreCLR", ".NET 9", "Resonite", "Thunderstore" ],
```

Comment:
I don't think you should have change netcore to net9, since net9 is part of net core and also I don't think thunderstore is a good classification here